src/FileGenerator/InvoicePdfFileGenerator.php

```diff
@@ -19,6 +19,11 @@
 
 final class InvoicePdfFileGenerator implements FileGeneratorInterface
 {
+    /**
+     * @var FilenameGeneratorInterface
+     */
+    protected $filenameGenerator;
```

Comment:
Please, mark it as private, the class is final, so there's no need to add protected fields to it.